app/src/features/surveys/view/SurveyHeader.tsx

```diff
@@ -128,6 +129,42 @@ const SurveyHeader: React.FC<ISurveyHeaderProps> = (props) => {
     });
   };
 
+  const showUploadSurveyDialog = () => {
+    dialogContext.setYesNoDialog({
+      dialogTitle: 'Upload Survey to Backbone',
```

Comment:
I would maybe just replace "Backbone" with "BioHub" anywhere the users can see it.  That seems to be the wording Robin has been using to describe what we think of as the Backbone. At least for now